crates/git_cmd/src/lib.rs

```diff
@@ -105,6 +105,22 @@ impl Repo {
         Ok(changed_files)
     }
 
+    pub fn changes_current_commit(
+        &self,
+        filter: impl FnMut(&&str) -> bool,
+    ) -> anyhow::Result<Vec<String>> {
+        let output = self.git(&[
+            "show",
+            "--oneline",
+            "--name-only",
+            "--pretty=format:''",
+            &self.current_commit_hash()?,
+            "-r",
```

Comment:
what's `-r`?
I tried this command with and without it and the result is the same

Reply:
I didn't find it in https://git-scm.com/docs/git-show

Reply:
its recursive for submodules

Reply:
https://stackoverflow.com/questions/424071/how-do-i-list-all-the-files-in-a-commit

Reply:
`-r` is related to `diff-tree` instead you used the `show` subcommand. Anyway I will switch to `diff-tree` because they recommend it 👍

Reply:
I think we can drop it because submodules would maybe have their ow release workflow